docs/version-history/v24-version.hist

```diff
@@ -19,6 +19,12 @@
   remote history queries will not have a negative impact on the *condor_schedd*
   :jira:`3215`
 
+- On Windows, :ad-attr:`ResidentSetSize` and thus :ad-attr:`MemoryUsage` will now
+  be updated as freqently as :ad-attr:`ImageSize` is so that the default
```

Comment:
```suggestion
- On Windows, :ad-attr:`ResidentSetSize` and thus :ad-attr:`MemoryUsage` will now
  be updated in the startd as frequently as :ad-attr:`ImageSize` is so that the default
```